actiontext/app/models/action_text/rich_text.rb

```diff
@@ -52,7 +52,7 @@ class RichText < Record
     has_many_attached :embeds
 
     before_save do
-      self.embeds = body.attachables.grep(ActiveStorage::Blob).uniq if body.present?
+      self.embeds = body.nil? ? [] : body.attachables.grep(ActiveStorage::Blob).uniq
```

Comment:
The first version :
```ruby
self.embeds = body.attachables.grep(ActiveStorage::Blob).uniq if body.present?
```
was not supporting nil.


I've changed it to account [for this comment](https://github.com/rails/rails/pull/51291#issuecomment-1987589730).

I agree that explicitly setting nil should empty embeds.